actin/operational/sync_actin_operations_vm

```diff
@@ -5,13 +5,27 @@ source locate_files || exit 1
 
 set -e
 
-info "Syncing ACTIN wgs pipeline output"
-gsutil -m rsync -r -x ".*\.bam$|.*\.cram$" $(locate_actin_wgs_pipeline_bucket) $(locate_actin_wgs_pipeline_output_directory)
-gsutil -m rsync -r -d -x ".*\.bam$|.*\.cram$" gs://actin-prod-wgs-pipeline-output-diagnostic/ /data/actin/wgs_pipeline_output_diagnostic
+actin_prod_project_id="actin-prod"
+project={$1:-$actin_prod_project_id}
 
-info "Syncing ACTIN patient data"
+info "Syncing ACTIN wgs pipeline output from GCP"
+gsutil -m rsync -r -x ".*\.bam$|.*\.cram$" $(locate_actin_wgs_pipeline_bucket $project) $(locate_actin_wgs_pipeline_output_directory)
+gsutil -m rsync -r -d -x ".*\.bam$|.*\.cram$" gs://$(project)-wgs-pipeline-output-diagnostic/ /data/actin/wgs_pipeline_output_diagnostic
+
+info "Syncing ACTIN input data from GCP"
+gsutil -m rsync -r -d $(locate_actin_clinical_feed_input_bucket $project) $(locate_actin_clinical_input_feed_directory)
+gsutil -m rsync -r -d $(locate_actin_clinical_curation_bucket $project) $(locate_actin_clinical_curation_directory)
+gsutil -m rsync -r -d $(locate_actin_trial_config_bucket $project) $(locate_actin_trial_config_directory)
+gsutil -m rsync -r -d $(locate_actin_ctc_config_bucket $project) $(locate_actin_ctc_config_directory)
+gsutil -m rsync -r -d $(locate_actin_trial_database_bucket $project) $(locate_actin_trial_database_directory)
+gsutil -m rsync -r -d $(locate_actin_analysis_bucket $project)/molecular/ $(locate_actin_molecular_directory)
+gsutil -m rsync -r -d $(locate_actin_analysis_bucket $project)/treatment_match/ $(locate_actin_treatment_match_directory)
+gsutil -m rsync -r -d $(locate_actin_analysis_bucket $project)/reports/ /data/actin/reports/
+
+info "Syncing ACTIN patient data to GCP"
```

Comment:
@spdeleeuw Do you know why we do this? I would have expected us to push data to the prod_data_bucket via the archive_after_sharing scripts. I would like to use the directories on the VM as an easy place to keep the latest outputs, but if I keep things this way risk overwriting what's in the share bucket.

TLDR; I'd like to remove this synchronization.

Reply:
Sorry Paul, but I honestly do not know what this script is used for. It's not part of regular production activities.